slides/k8s/configuration.md

```diff
@@ -0,0 +1,529 @@
+# Managing configuration
+
+- Some applications need to be configured (obviously!)
+
+- There are many ways for our code to pick up configuration:
+
+  - command-line arguments
+
+  - environment variables
+
+  - configuration files
+
+  - configuration servers (getting configuration from a database, an API...)
+
+  - ... and more (because programmers can be very creative!)
+
+- How can we fit that with containers and Kubernetes?
```

Comment:
I'm not clear on what "fit that" is referring to. Do you mean "use this with"?

Reply:
I'll go for `s/fit that/do these things/`